bin/phase-helpers.sh

```diff
@@ -696,7 +696,7 @@ einstall() {
 		unset EI_DESTLIBDIR
 	fi
 
-	if [ -f ./[mM]akefile -o -f ./GNUmakefile ] ; then
+	if [ -f ./Makefile -o -f ./makefile -o -f ./GNUmakefile ] ; then
```

Comment:
Don't use `[ ... ]` in bash, it's the source of issues like the one you're trying to fix.

Reply:
The use of `[` or `[[` doesn't seem to matter. I can't get a similar test to work:
```sh
touch a ab ac
[ -f "./a[bc]" ] && echo 'true'
[[ -f "./a[bc]" ]] && echo 'true'
```
Produces no output.

The test can be changed to `[[` for consistency. I left it as it was because I suspected it was chosen for a reason. That reason may have been the test did not obviously fail, per the `shellcheck` Wiki.

Reply:
I've just tested with with app-shells/bash-4.4_p12, and `[ -f ./[mM]akefile ]` works with the single brackets, while `[[ -f ./[mM]akefile ]]` does not. So that explains the use of single brackets.

However, if both files exist then `[ -f ./[mM]akefile ]` fails:
```shell
$ [ -f ./[mM]akefile ]
bash: [: ./makefile: binary operator expected
$ echo $?
2
```

Reply:
Fixed in 46324633cbcf50fa6e09bedfe317f088de50fc08.